assets/stylesheets/_sections.scss

```diff
@@ -110,61 +140,160 @@
 }
 .leaderboard__item {
   font-family: $font-stack--retro;
-  font-size: $type--large;
+  font-size: $type--medium;
   color: $white;
+  @media screen and (min-width: $bp-small+100){
+    font-size: $type--large;
+  }
+}
+
+
+.blink{
+  color:$lemon;
+  -webkit-animation: blink linear 1.25s infinite;
+  animation: blink linear 1.25s infinite;
+}
+@-webkit-keyframes blink {
+  0% { opacity: 1; } 
+  50% { opacity: .1; } 
+  100% { opacity: 1; }
+}
+@keyframes blink {
+  0% { opacity: 1; } 
+  50% { opacity: .1; } 
+  100% { opacity: 1; }
 }
 
 .arcade-section__teaser {
   font-family: $font-stack--retro;
   font-size: $type--small;
   color: $white;
+  text-align: center;
 }
 
-.events-section {
-  align-items: center;
-  display: flex;
+.events-section{
+  background-color: #020710;
```

Comment:
(i see this raw hex 😜)

Reply:
oh ya hahah i should probably update that lol $variables-only